mx.truffleruby/suite.py

```diff
@@ -296,9 +296,9 @@
 
         # Set of extra files to extract to run Ruby
         "TRUFFLERUBY-ZIP": {
-            "native": True, # Not Java
+            "native": True,  # Not Java
             "relpath": True,
-            "platformDependent": True, # truffleruby-cext
+            "platformDependent": True,  # truffleruby-cext
```

Comment:
Ah did not notice, I though it's missing a space. Found a way to reconfigure the autformatter, I'll fix